tests/tracer/test_logging.py

```diff
@@ -0,0 +1,202 @@
+import os
+
+
+def test_unrelated_logger_in_debug_with_patch_all(run_python_code_in_subprocess):
+    """
+    When importing the tracer after the custom log,
+                the ddtrace logger does not override any custom logs settings
+    """
+    env = os.environ.copy()
+    env["DD_TRACE_DEBUG"] = "true"
+    env["DD_TRACE_LOG_FILE"] = "./testlog.log"
```

Comment:
We should use the [tmpdir](https://docs.pytest.org/en/latest/reference/reference.html#std-fixture-tmpdir) pytest fixture here so we can get an isolated temporary folder for each test case.


example (untested) usage:

```python
def test_my_test(tmpdir):
    log_file = tmp_dir += "/testlog.log"
    env["DD_TRACE_LOG_FILE"] = log_file

    out, err, status, pid = run_python_code_in_subprocess()
    assert status == 0, err
    assert err == b""
    assert out == b""

    with open(log_file) as fp:
        log_contents = fp.read()
        assert len(log_contents)
```

Reply:
The current version of the tests use `tmp_dir` so I am marking this as resolved!